tests/integration/sites_test.py

```diff
@@ -49,38 +49,28 @@ def test_all_transitions(self, vasp_sites):
         transitions = vasp_sites.transitions
 
         events = transitions.events
-        assert transitions.events.shape == (450, 5)
+        assert transitions.events.shape == (2105, 6)
```

Comment:
Why did the number of events change by so much?

Reply:
because transitions are now _all_ transitions, not only jumps

Reply:
I would be more confident in these changes if we could reproduce the 450 number here (maybe a separate test).

Reply:
then it should be `jumps.jumps.shape == 462` (that is already tested somewhere else). There are a bit more jumps, because the old method _always_ throws the last jump from an atom away as bad, while it is only bad if it happens on the last timestep.